esp/src/package.json

```diff
@@ -30,57 +30,58 @@
     "rm-hpcc": "rimraf ./node_modules/@hpcc-js",
     "start": "webpack-dev-server -d --env development",
     "test": "run-s lint",
-    "update": "npx npm-check-updates -u"
+    "update": "npx npm-check-updates -u -t minor"
   },
   "main": "src/stub.js",
   "dependencies": {
-    "@fluentui/react": "^8.49.7",
+    "@fluentui/react": "^8.56.0",
     "@fluentui/react-cards": "1.0.0-beta.0",
-    "@fluentui/react-hooks": "^8.3.9",
-    "@fluentui/react-icons-mdl2": "^1.2.12",
-    "@hpcc-js/chart": "^2.69.0",
-    "@hpcc-js/codemirror": "^2.52.0",
-    "@hpcc-js/common": "^2.59.0",
-    "@hpcc-js/comms": "2.64.0",
+    "@fluentui/react-hooks": "^8.3.13",
+    "@fluentui/react-icons-mdl2": "^1.2.16",
+    "@fluentui/react-experiments": "^8.4.34",
+    "@hpcc-js/chart": "^2.74.0",
+    "@hpcc-js/codemirror": "^2.57.0",
+    "@hpcc-js/common": "^2.64.0",
+    "@hpcc-js/comms": "2.69.0",
     "@hpcc-js/dataflow": "^7.0.0",
-    "@hpcc-js/eclwatch": "^2.64.0",
-    "@hpcc-js/graph": "^2.72.0",
-    "@hpcc-js/html": "^2.34.0",
-    "@hpcc-js/layout": "^2.39.0",
-    "@hpcc-js/map": "^2.67.0",
-    "@hpcc-js/other": "^2.13.72",
-    "@hpcc-js/phosphor": "^2.14.53",
-    "@hpcc-js/react": "^2.42.0",
-    "@hpcc-js/tree": "^2.32.0",
-    "@hpcc-js/util": "^2.40.0",
-    "clipboard": "^2.0.8",
+    "@hpcc-js/eclwatch": "^2.70.0",
+    "@hpcc-js/graph": "^2.77.0",
+    "@hpcc-js/html": "^2.39.0",
+    "@hpcc-js/layout": "^2.45.0",
+    "@hpcc-js/map": "^2.73.0",
+    "@hpcc-js/other": "^2.13.78",
+    "@hpcc-js/phosphor": "^2.14.59",
+    "@hpcc-js/react": "^2.47.0",
+    "@hpcc-js/tree": "^2.37.0",
+    "@hpcc-js/util": "^2.44.0",
+    "clipboard": "^2.0.10",
     "detect-browser": "^5.3.0",
-    "dijit": "^1.16.4",
-    "dijit-themes": "^1.16.4",
-    "dojo": "^1.16.4",
-    "dojo-util": "^1.16.4",
-    "dojox": "^1.16.4",
+    "dijit": "^1.17.0",
+    "dijit-themes": "^1.17.0",
+    "dojo": "^1.17.0",
+    "dojo-util": "^1.17.0",
+    "dojox": "^1.17.0",
     "es6-promise": "^4.2.8",
     "font-awesome": "^4.7.0",
     "formik": "^2.2.9",
-    "query-string": "^7.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-hook-form": "^7.24.1",
-    "react-reflex": "^4.0.3",
+    "query-string": "^7.1.1",
+    "react": "^16.14.0",
```

Comment:
Did this cause problems when it was bumped to 17 in https://github.com/hpcc-systems/HPCC-Platform/pull/15637? I hadn't noticed any.

Reply:
@fluent issues a sibling warning.